docs/developer-guide/etendo-classic/bundles/platform-extensions-bundle.md

```diff
@@ -202,3 +202,76 @@ Then `./gradlew setup` must be executed.
 !!! warning
     It is mandatory to execute this step before installing the module. 
 
+
+## EAN 128
+
+### Overview
```

Comment:
All modules have their Technical Aspects description and in this case only Overview and description of methods or classes are added.